nb2/service/person_service.py

```diff
@@ -48,21 +77,21 @@ def get_person_name_by_slack_user_id(slack_user_id: str):
         return person.first_name
 
 
-def get_random_person():
+def get_random_person() -> str:
     """
     Return random Person's slack_user_id.
 
     Returns:
         If there are any Persons in the system, return the slack_user_id
-        of one of them, otherwise return None.
+        or ghost_user_id of one of them, otherwise return None.
     """
     person = Person.query.order_by(func.random()).first()
 
     if person is not None:
-        return person.slack_user_id
+        return person.slack_user_id or person.ghost_user_id
 
 
-def create_person(data: CreatePersonDTO):
+def create_person(data: Union[CreatePersonDTO, CreateGhostPersonDTO]):
```

Comment:
So this allows us to create new quotes for ghost users, is this what we'd like to allow? I understand adding quotes with a ghost_id so that we can remember them if they leave but I don't think we should allow adding quotes for people that have left. Correct me if I'm wrong though

Reply:
I vaguely remember one of the motivations for this was because people like quoting random things (that aren't people) or people that were never in the SC workspace to begin with...

Reply:
Yup! I think we have some quotes from Jay for example. And some SC alums will join events/meetups and could say quotable things.

Also, Hi 👋  Happy to see you both still working on this ❤️

Reply:
hi Kelly :wave: :partying_face: